app/api/dependencies.py

```
@@ -91,6 +108,70 @@
+    serving the old status until the process restarted.
+    """
+    app.state.cache.beatmapset.pop(set_id, None)
+    for map_id in map_ids:
+        app.state.cache.beatmap.pop(map_id, None)
+    for map_md5 in map_md5s:
+        app.state.cache.beatmap.pop(map_md5, None)
+
+
+def _utc_now() -> datetime:
+    """The current time, as the submission pipeline stamps it."""
+    return datetime.now()
```

Ruff (0.16.2)

[warning] `datetime.datetime.now()` called without a `tz` argument

(DTZ005)